cli/commands.c

```diff
@@ -370,7 +369,7 @@ int login(int argc, char **argv) {
     return EXIT_FAILURE;
   }
 
-  char *path = strstr(argv[optind], "/v1/");
+  char *path = strstr(argv[optind], "v1/");
```

Comment:
Are you worried about breaking the cli? Maybe now, before the 1.0 release, would be a good time to do so.

Reply:
My test case from July still breaks ;-) But I'll leave it at that, because the happy path (un-prefixed challenge) is working fine, and the fact that the cli also supports most URLs is a happy coincidence. We should reconsider this for 1.0 because of Hyrum's Law, though.